conda_forge_tick/migrators.xsh

```diff
@@ -1322,16 +1321,18 @@ class MigrationYaml(Migrator):
     """Migrator for bumping the build number."""
     migrator_version = 0
     rerender = True
-    bump_number = 0
```

Comment:
This means: default is to bump unless declared, right? If so bumping is a True/False or an acutal value to bump? Like,

```
  build_number:
    1
```

makes me think that is +1 to the build number, right?

Reply:
yes, `1` means +1